numalogic/config/_config.py

```diff
@@ -34,6 +34,21 @@ class ModelInfo:
 
 
 # TODO add this in the right config
+@dataclass
+class JitterConf:
```

Comment:
Nitpick: a generic JitterConf doesn't need to hardcode a (physical) dimension. This actually is a TimeJitterConf.

Another nitpick: jitter_secs is plural, jitter_steps_min is singular, model_expiry_sec is singular, retry_secs is plural.
Again, it's a nitpick so no need to change those terms in this PR. but it's one aspect to pay attention to in future PRs.

Reply:
Let me add a TODO for this.